script/prerender/contentPages.tsx

```diff
@@ -31,12 +31,39 @@ export const stats = {
   renderHtml: 0,
 };
 
-export async function prepareContentPage(
+export type BookMatch = {
+  params: { book: { slug: string } },
+  route: typeof content,
+  state: { bookUid: string, bookVersion: string },
+};
+export type SerializedBookMatch = Omit<BookMatch, 'route'>;
+
+export function deserializeBook(book: SerializedBookMatch) {
```

Comment:
I did this at first but then it turned out to not be necessary. I could just delete `deserializeBook()`, since the sitemapIndex stuff did not need the `route: content`.